doc/modules/model_evaluation.rst

```diff
@@ -215,6 +215,25 @@ the following two rules:
   Again, by convention higher numbers are better, so if your scorer
   returns loss, that value should be negated.
 
+.. notes:: **Using custom scorers in functions where `n_jobs > 1`**
+
```

Comment:
You need to indent the whole block for it to be included in the notes section.

Reply:
Fixed it. Also, able to view the documentation locally now; shouldn't happen again. Apologies.